sw/vendor/README.md

```diff
@@ -0,0 +1,14 @@
+# Work with software code in external repositories
+
+Please refer to [vendor_hw](../../doc/vendor_hw.md) for more details on rationale of vendoring external repositories
+
+
+## RISCV_COMPLIANCE
+
+RISCV_COMPLIANCE defines a series of tests that to ensure the riscv core is compliant.
```

Comment:
RISCV_COMPLIANCE -> "The RISC-V Compliance Suite"
riscv core -> RISC-V system
compliant -> complies with the RISC-V Specification.

Reply:
fixed.